test/core/event_engine/test_suite/timer_test.cc

```diff
@@ -141,7 +141,7 @@ void EventEngineTimerTest::ScheduleCheckCB(
 
 TEST_F(EventEngineTimerTest, StressTestTimersNotCalledBeforeScheduled) {
   auto engine = this->NewEventEngine();
-  constexpr int thread_count = 100;
+  constexpr int thread_count = 10;
   constexpr int call_count_per_thread = 100;
```

Comment:
I imagine overall system load is a problem, not just threads, but on the off chance you think otherwise: we could consider bumping up the `call_count_per_thread` to increase the stress on EventEngine without additional threads.

Reply:
It probably is... this is a first cut. I think we want thread count <= 10 wherever possible (just for resource usage), but yes we could increase the call count.

A good way of profiling is to run:

`bazel run test/core/path/to:test --run_under='time -p'`

average cpu utilization for the run will be `(user+system)/wall`